contents/docs/_snippets/identify-how-it-works.mdx

```diff
@@ -1,11 +1,15 @@
-When a user starts browsing your website or app, PostHog automatically assigns them an **anonymous ID**, which is stored locally. This enables us to track anonymous users – even across different sessions.
+When a user starts browsing your website or app, PostHog automatically assigns them an **anonymous ID**, which is stored locally. 
 
-> **Note:** depending on your [persistence configuration](/docs/libraries/js/persistence), the anonymous ID may not be stored, and thus regenerated across sessions.
+Provided you've [configured persistence](/docs/libraries/js/persistence) to use cookies or `localStorage`, this enables us to track anonymous users – even across different sessions.
 
 By calling `identify` with a `distinct_id` of your choice (usually the user's ID in your database, or their email), you link the anonymous ID and distinct ID together.
 
-Thus all past and future events made with that anonymous ID are now associated with the distinct ID. This enables you to do things like associate events with a user from before they log in for the first time, or associate their events across different devices or platforms.
+Thus all past and future events made with that anonymous ID are now associated with the distinct ID. 
 
-> **Using `identify` in the backend:**
->
-> Although it is technically possible to call `identify` using our backend SDKs, it is typically most useful in frontends. This is because there is no concept of anonymous sessions in the backend SDKs.
\ No newline at end of file
+This enables you to do things like associate events with a user from before they log in for the first time, or associate their events across different devices or platforms.
+
+<details>
+  <summary>Using `identify` in the backend</summary>
+
+  Although it is technically possible to call `identify` using our backend SDKs, it is typically most useful in frontends. This is because there is no concept of anonymous sessions in the backend SDKs.
```

Comment:
```suggestion
  Although you can call `identify` using our backend SDKs, it is used most in frontends. This is because there is no concept of anonymous sessions in the backend SDKs, so calling `identify` only updates person profiles. 
```